.github/workflows/sonar.yaml

```diff
@@ -1,4 +1,6 @@
 name: SonarCloud Build
+env:
```

Comment:
We have showcase version here and in [ci-maven.yaml](https://github.com/googleapis/gapic-generator-java/blob/79d986bd35b49a819dc875cfec69ec8685517930/.github/workflows/ci-maven.yaml#L8)as well, I'm not very familiar with Github actions, is it possible to configure a shared `SHOWCASE_VERSION`?

Reply:
If we are going to have hermetic builds, the version needs to be stored in source.